storage/rocksdb/ha_rocksdb.h

```diff
@@ -266,6 +269,8 @@ class ha_rocksdb : public my_core::handler, public blob_buffer {
   /* Type of locking to apply to rows */
   Rdb_lock_type m_lock_rows;
 
+  bool m_use_range_locking;
```

Comment:
It seems to me like this belongs more on the `Rdb_transaction` object than on `ha_rocksdb` (whereas `m_lock_rows` may be different per table in the same transaction).